cypress/e2e/api/health-check-dms.cy.ts

```diff
@@ -0,0 +1,96 @@
+import { container } from '@utils';
+import { CreateStoreScenario, UserLoginScenario } from '@scenarios/backoffice';
+import { HealthCheckDmsStaticFixtures } from '@interfaces/api';
+import { HealthCheckDmsDynamicFixtures } from "../../support/types/api";
+
+(Cypress.env('isDynamicStoreEnabled') ? describe : describe.skip)('health check dms', { tags: '@dms' }, () => {
+  const userLoginScenario = container.get(UserLoginScenario);
+  const createStoreScenario = container.get(CreateStoreScenario);
+
+  let staticFixtures: HealthCheckDmsStaticFixtures;
+  let dynamicFixtures: HealthCheckDmsDynamicFixtures;
+
+  before(function () {
+
+    ({ staticFixtures, dynamicFixtures } = Cypress.env());
+    createNewStoreInBackoffice();
+    cy.request({
+      method: 'POST',
+      url: Cypress.env().glueBackendUrl + '/token',
+      headers: {
+        'Content-Type': 'application/x-www-form-urlencoded',
+      },
+      body: `grantType=password&username=admin@spryker.com&password=${staticFixtures.defaultPassword}`,
```

Comment:
Email is hardcoded. Previously dynamicFitures.rootUser.username was used, but auth was not successful. I tried to fix it and saw that it requires some time for me. Maybe you know how to do it fast, @dmiseev